### PR TITLE
Feature/implement api

### PR DESCRIPTION
**Pull request by Lordkaito**
In this branch:
1. Use API calls to build the leaderboard
2. Fetch data from API and parse it using JSON methods.
3. Implement refresh button
4. Implement submit button

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -18,25 +18,12 @@ <h1>
     <div id="table-leaderboard">
       <div class="btn-container">
         <h3>Recent scores</h3>
-        <input type="button" value="Refresh">
+        <input type="button" value="Refresh" id="refresh">
       </div>
       <div id="points">
-        <div id="scores">
+        <!-- <div id="scores">
           <div class="score-name">My Name 1</div>
```
Comment (Dipeshtwis) [inline src/index.html]: - [x] I would recommend removing the commented code to makes the codebase cleaner.

File: src/api.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  const response = await fetch(gameUrl, {
+    method: 'POST',
+    body: JSON.stringify({ name: newGameName }),
+    headers: {
+      'Content-Type': 'application/json; charset=utf-8',
+    },
+  });
+  const jsonData = await response.json();
+  return jsonData;
+};
```
Comment (Dipeshtwis) [inline src/api.js:15]: [OPTIONAL]

- I would appreciate your work and you've done really good but this is not required for this application as you are not using this code in your application, you can create a new game with this but there is no need here. So putting this in your codebase is not necessary and I would recommend to remove this, again this is optional so it's up to you. :smile:

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Lordkaito, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Used `async` and `await` with the `fetch` method to send and receive data.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**_Check the comments under the review_**

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rukundoeric — APPROVED** [review]
Hi @Lordkaito,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**